CHANGELOG.md

```diff
@@ -1,8 +1,16 @@
 # Changelog
 
-## [1.0](https://github.com/rapidsai/gQuant/tree/1.0) (2020-12-17)
+## [1.0.1](https://github.com/rapidsai/gQuant/tree/1.0.1) (2021-01-20)
 
-[Full Changelog](https://github.com/rapidsai/gQuant/compare/0.5...1.0)
+[Full Changelog](https://github.com/rapidsai/gQuant/compare/v1.0.0...1.0.1)
```

Comment:
Will the version be tagged with or without "v" as `1.0.1` or as `v1.0.1`. If with "v" update the changelog to use "v1.0.1".

Reply:
good catch. I added the 'v'